Libraries/Lists/VirtualizedList.js

```diff
@@ -1331,9 +1331,21 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     const preferredScrollerStyleDidChangeHandler =
       this.props.onPreferredScrollerStyleDidChange;
 
+    const isFirstRowSelected = this.state.selectedRowIndex === this.state.first;
+    const isLastRowSelected = this.state.selectedRowIndex === this.state.last;
+
+    // Don't pass in ArrowUp/ArrowDown at the top/bottom of the list so that keyboard event can bubble
```

Comment:
What does "keyboard event can bubble" mean

Reply:
One day I'll do a brown bag about this, but for now:

https://javascript.info/bubbling-and-capturing